src/attached-file/remove-attached-file.js

```diff
@@ -0,0 +1,19 @@
+import Command from "@ckeditor/ckeditor5-core/src/command";
+
+export default class RemoveAttachedFile extends Command {
+    execute(options) {
+        const { id } = options || { id: '' };
```

Comment:
as we discussed we have to exit if `id` param does not exist.

otherwise you will try to find id with empty string value, that does not make sense